vcpkg/consume/third-party-authentication.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Authenticate to third-party services"
+description: Explain authentication methods for third-party services consumed by vcpkg
+author: vicroms
+ms.author: viromer
+ms.date: 03/03/2025
+ms.topic: tutorial
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+---
+# Authenticate to third-party services
```

Comment:
Consider breaking this into multiple separate articles, because it seems like this article is serving disjoint audiences.